### PR TITLE
[zh-cn] Localize blog: 2024-07-19-kubernetes-1.31-deprecations-and-removals.md

### PR DESCRIPTION
**Pull request by my-git9**
Localize blog: 2024-07-19-kubernetes-1.31-deprecations-and-removals.md

### CODE REVIEW

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+CephFS volume plugin was formally marked as deprecated in v1.28.
+-->
+### 删除 CephFS 卷插件
+
+[CephFS 卷插件](/docs/concepts/storage/volumes/#cephfs) 已在此版本中删除，
+并且 `cephfs` 卷类型已无法使用。
+
+建议你改用 [CephFS CSI 驱动程序](https://github.com/ceph/ceph-csi/) 作为第三方存储驱动程序。
+如果你在将集群版本升级到 v1.31 之前使用 CephFS 卷插件，则必须重新部署应用程序才能使用新驱动程序。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:197]:
```suggestion
如果你在将集群版本升级到 v1.31 之前在使用 CephFS 卷插件，则必须重新部署应用才能使用新驱动。
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+The v1.31 release will remove the [Ceph RBD volume plugin](/docs/concepts/storage/volumes/#rbd) and its CSI migration support, making the `rbd` volume type non-functional.
+
+It's recommended that you use the [RBD CSI driver](https://github.com/ceph/ceph-csi/) in your clusters instead. 
+If you were using Ceph RBD volume plugin before upgrading the cluster version to v1.31, you must re-deploy your application to use the new driver.
+
+The Ceph RBD volume plugin was formally marked as deprecated in v1.28.
+-->
+### 删除 Ceph RBD 卷插件
+
+v1.31 版本将删除 [Ceph RBD 卷插件](/docs/concepts/storage/volumes/#rbd) 及其 CSI 迁移支持，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:213]:
```suggestion
v1.31 版本将删除 [Ceph RBD 卷插件](/zh-cn/docs/concepts/storage/volumes/#rbd) 及其 CSI 迁移支持，
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+`rbd` 卷类型将无法继续使用。
+
+建议你在集群中使用 [RBD CSI 驱动程序](https://github.com/ceph/ceph-csi/)。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:216]:
```suggestion
建议你在集群中使用 [RBD CSI 驱动](https://github.com/ceph/ceph-csi/)。
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+如果你在将集群版本升级到 v1.31 之前使用 Ceph RBD 卷插件，则必须重新部署应用程序以使用新驱动程序。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:217]:
```suggestion
如果你在将集群版本升级到 v1.31 之前在使用 Ceph RBD 卷插件，则必须重新部署应用以使用新驱动。
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+It's recommended that you use the `NodeVolumeLimits` plugin instead because it can handle the same functionality as the removed plugins since those volume types have been migrated to CSI. 
+Please replace the deprecated plugins with the `NodeVolumeLimits` plugin if you explicitly use them in the [scheduler config](/docs/reference/scheduling/config/). 
+The `AzureDiskLimits`, `CinderLimits`, `EBSLimits`, and `GCEPDLimits` plugins will be removed in a future release.
+
+These plugins will be removed from the default scheduler plugins list as they have been deprecated since Kubernetes v1.14.
+-->
+建议你改用 `NodeVolumeLimits` 插件，因为它可以处理与已删除插件相同的功能，因为这些卷类型已迁移到 CSI。
+如果你在[调度器配置](/docs/reference/scheduling/config/)中明确使用已弃用的插件，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:245]:
```suggestion
如果你在[调度器配置](/zh-cn/docs/reference/scheduling/config/)中显式使用已弃用的插件，
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+
+* The `flowcontrol.apiserver.k8s.io/v1beta3` API version of FlowSchema and PriorityLevelConfiguration will be removed. 
+To prepare for this, you can edit your existing manifests and rewrite client software to use the `flowcontrol.apiserver.k8s.io/v1 API` version, available since v1.29. 
+All existing persisted objects are accessible via the new API. Notable changes in flowcontrol.apiserver.k8s.io/v1beta3 include that the PriorityLevelConfiguration `spec.limited.nominalConcurrencyShares` field only defaults to 30 when unspecified, and an explicit value of 0 is not changed to 30.
+
+For more information, please refer to the [API deprecation guide](/docs/reference/using-api/deprecation-guide/#v1-32).
+-->
+## 展望未来
+
+[Kubernetes v1.32](/docs/reference/using-api/deprecation-guide/#v1-32) 计划删除的官方 API 列表包括：
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:263]:
```suggestion
[Kubernetes v1.32](/docs/reference/using-api/deprecation-guide/#v1-32) 计划删除的官方 API 包括：
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+
+* 将删除 `flowcontrol.apiserver.k8s.io/v1beta3` API 版本的 FlowSchema 和 PriorityLevelConfiguration。
+为了做好准备，你可以编辑现有清单并重写客户端软件以使用自 v1.29 起可用的 `flowcontrol.apiserver.k8s.io/v1 API` 版本。
+所有现有的持久化对象都可以通过新 API 访问。`flowcontrol.apiserver.k8s.io/v1beta3` 中需要注意的变化包括优先级配置
+`spec.limited.nominalConcurrencyShares` 字段仅在未指定时默认为 30，并且显式设置为 0 的话不会被更改为 30。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:268]:
```suggestion
  为了做好准备，你可以编辑现有清单并重写客户端软件以使用自 v1.29 起可用的 `flowcontrol.apiserver.k8s.io/v1 API` 版本。
  所有现有的持久化对象都可以通过新 API 访问。`flowcontrol.apiserver.k8s.io/v1beta3` 中需要注意的变化包括优先级配置
  `spec.limited.nominalConcurrencyShares` 字段仅在未指定时默认为 30，并且显式设置为 0 的话不会被更改为 30。
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+<!--
+## Want to know more?
+The Kubernetes release notes announce deprecations. 
+We will formally announce the deprecations in [Kubernetes v1.31](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md#deprecation) as part of the CHANGELOG for that release.
+
+You can see the announcements of pending deprecations in the release notes for:
+-->
+## 想要了解更多？
+
+Kubernetes 发行说明宣布弃用。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:281]:
```suggestion
Kubernetes 发行说明中会宣布弃用信息。
```

File: content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+我们将在 [Kubernetes v1.31](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md#deprecation)
+中正式宣布弃用，作为该版本的 CHANGELOG 的一部分。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-07-19-kubernetes-1.31-deprecations-and-removals.md:283]:
```suggestion
中正式宣布弃用信息，作为该版本的 CHANGELOG 的一部分。
```

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**